packages/frontend/src/components/chart/Description.tsx

```diff
@@ -1,10 +1,15 @@
 import React from 'react'
 
-export function Description() {
+export interface DescriptionProps {
+  hidden: boolean
+}
+
+export function Description({ hidden }: DescriptionProps) {
   return (
     <p
       data-role="chart-description"
       className="col-start-1 sm:col-start-2 col-span-4 sm:col-span-2 text-center text-sm flex justify-center"
+      style={hidden ? { display: 'none' } : {}}
```

Comment:
It would be cleaner to add a new class using classnames and use tailwind's `hidden`.